Makefile

```diff
@@ -1,67 +1,86 @@
-IOS_PLATFORM_DEVELOPER = ${XCODE_DEVELOPER}/Platforms/${IOS_PLATFORM}.platform/Developer
-IOS_SDK = ${IOS_PLATFORM_DEVELOPER}/SDKs/$(shell ls ${IOS_PLATFORM_DEVELOPER}/SDKs | sort -r | head -n1)
+PLATFORM_DEVELOPER = ${XCODE_DEVELOPER}/Platforms/${PLATFORM}.platform/Developer
+SDK = ${PLATFORM_DEVELOPER}/SDKs/$(shell ls ${PLATFORM_DEVELOPER}/SDKs | sort -r | head -n1)
 
 all: lib/libspatialite.a
 lib/libspatialite.a: build_arches
 	mkdir -p lib
 	mkdir -p include
 
 	# Copy includes
-	cp -R build/arm64/include/geos include
-	cp -R build/arm64/include/spatialite include
-	cp -R build/arm64/include/*.h include
+	cp -R build/iPhoneOS/arm64/include/geos include
+	cp -R build/iPhoneOS/arm64/include/spatialite include
+	cp -R build/iPhoneOS/arm64/include/*.h include
 
-	# Make fat libraries for all architectures
-	for file in build/arm64/lib/*.a; \
+	# Make fat libraries for iOS
+	for file in build/iPhoneOS/arm64/lib/*.a; \
 		do name=`basename $$file .a`; \
 		lipo -create \
-			-arch armv7 build/armv7/lib/$$name.a \
-			-arch armv7s build/armv7s/lib/$$name.a \
-			-arch arm64 build/arm64/lib/$$name.a \
-			-arch i386 build/i386/lib/$$name.a \
-			-arch x86_64 build/x86_64/lib/$$name.a \
+			-arch armv7 build/iPhoneOS/armv7/lib/$$name.a \
+			-arch armv7s build/iPhoneOS/armv7s/lib/$$name.a \
+			-arch arm64 build/iPhoneOS/arm64/lib/$$name.a \
+			-arch arm64e build/iPhoneOS/arm64e/lib/$$name.a \
+			-arch i386 build/iPhoneSimulator/i386/lib/$$name.a \
+			-arch x86_64 build/iPhoneSimulator/x86_64/lib/$$name.a \
 			-output lib/$$name.a \
 		; \
 		done;
 
+	# Make fat libraries for watchOS
+	for file in build/watchOS/armv7k/lib/*.a; \
+		do name=`basename $$file .a`; \
+		lipo -create \
+			-arch armv7k build/watchOS/armv7k/lib/$$name.a \
+			-arch arm64_32 build/watchOS/arm64_32/lib/$$name.a \
+			-arch i386 build/watchSimulator/i386/lib/$$name.a \
+			-arch x86_64 build/watchSimulator/x86_64/lib/$$name.a \
+			-output lib/$${name}_watchos.a \
+		; \
+		done;
+
 # Build separate architectures
 build_arches:
-	${MAKE} arch ARCH=armv7 IOS_PLATFORM=iPhoneOS HOST=arm-apple-darwin
-	${MAKE} arch ARCH=armv7s IOS_PLATFORM=iPhoneOS HOST=arm-apple-darwin
-	${MAKE} arch ARCH=arm64 IOS_PLATFORM=iPhoneOS HOST=arm-apple-darwin
-	${MAKE} arch ARCH=i386 IOS_PLATFORM=iPhoneSimulator HOST=i386-apple-darwin
-	${MAKE} arch ARCH=x86_64 IOS_PLATFORM=iPhoneSimulator HOST=x86_64-apple-darwin
-
-PREFIX = ${CURDIR}/build/${ARCH}
+	${MAKE} arch ARCH=armv7k PLATFORM=watchOS HOST=arm-apple-darwin MIN_VERSION=watchos-version-min=3.0
+	${MAKE} arch ARCH=arm64_32 PLATFORM=watchOS HOST=arm-apple-darwin MIN_VERSION=watchos-version-min=3.0
+	${MAKE} arch ARCH=i386 PLATFORM=watchSimulator HOST=i386-apple-darwin MIN_VERSION=watchos-version-min=3.0
+	${MAKE} arch ARCH=x86_64 PLATFORM=watchSimulator HOST=x86_64-apple-darwin MIN_VERSION=watchos-version-min=3.0
+	${MAKE} arch ARCH=armv7 PLATFORM=iPhoneOS HOST=arm-apple-darwin MIN_VERSION=iphoneos-version-min=7.0
```

Comment:
Does this overwrite `MIN_VERSION` defined above? Can/should we raise it to 9?

Reply:
Yep, this overwrites the default above. For raising: See above.